packages/mui-material/src/Radio/Radio.js

```diff
@@ -34,7 +34,11 @@ const RadioRoot = styled(SwitchBase, {
   overridesResolver: (props, styles) => {
     const { ownerState } = props;
 
-    return [styles.root, styles[`color${capitalize(ownerState.color)}`]];
+    return [
+      styles.root,
+      styles[ownerState.size !== 'medium' && `size${capitalize(ownerState.size)}`],
```

Comment:
This works. But it should be:
```suggestion
      ownerState.size !== 'medium' && styles[`size${capitalize(ownerState.size)}`],
```
so that `styles[false]` isn't returned when the condition is false. Same is done in other places.